### DISCUSSION TITLE
Add a way to invalidate cache for all loaders at once.

### DISCUSSION THREAD

**Discussion by minht11** [posva/unplugin-vue-router#208] [Ideas]
Currently there it is possible to invalidate cache of an individual loader, but no way to do that for all of them at once.

### Use case
User logs out, loaders are cached, another user logs in. To prevent new user seeing previous user data, we need to invalidate previous user loaders, but doing it manually is cumbersome, needing to account for all loaders.

### Proposed solution
Add a way to invalidate cache for all loaders at once.

**Comment by posva** [comment]
I think this is a good idea. I will need to explore more of the cache invalidation.

**Comment by posva** [comment]
Since cache was moved to pinia colada, this is possible with pinia colada instead
